appshell/appshell_extensions_mac.mm

```diff
@@ -1013,12 +1013,29 @@ NSUInteger processKeyString(ExtensionString& key)
         appshell_extensions::fixupKey(key, "Opt-", "")) {
         mask |= NSAlternateKeyMask;
     }
+
+    unichar pageUpChar   = NSPageUpFunctionKey;
+    unichar pageDownChar = NSPageDownFunctionKey;
+    unichar homeChar     = NSHomeFunctionKey;
+    unichar endChar      = NSEndFunctionKey;
+    unichar insertChar   = NSHelpFunctionKey;
+    
     //replace special keys with ones expected by keyEquivalent
-    const ExtensionString del = (ExtensionString() += NSDeleteCharacter);
+    const ExtensionString pageUp    = (ExtensionString() += [[NSString stringWithCharacters: &pageUpChar length: 1] UTF8String]);
```

Comment:
@rlim Why can't the new keys be defined using same pattern as existing keys? For example:

```js
    const ExtensionString pageUp    = (ExtensionString() += NSPageUpFunctionKey);
```

Seems simpler and more consistent.